### ISSUE TITLE
Include a comparison to django-redis-cache in the readme

### ISSUE THREAD

**Issue by edmorley** [niwinz/django-redis#217]
An alternative Django redis package is django-redis-cache:
https://github.com/sebleier/django-redis-cache
http://django-redis-cache.readthedocs.io/en/latest/

To assist with people trying to decide between them, it would be good to have a quick comparison either in the readme or the main documentation :-)

(My searches only found http://stackoverflow.com/questions/21932097/difference-between-django-redis-cache-and-django-redis-for-redis-caching-with-dj)

**Comment by niwinz** [comment]
Glad to accept a PR for that ;)

**Comment by malefice** [comment]
I agree with what the stackoverflow post said. In addition, I think `django-redis-cache` is more suitable for simpler use cases, while `django-redis` is more for people who would actually need raw access.

On a more personal note, I chose to use `django-redis-cache` months ago, because my use case is not complex, and I was relatively new to Django and much newer to Redis. In that situation, the difference in how the docs were written was a significant factor in my decision.

For example, it is kind of weird that this project's documentation has an "Advanced Usage" section, but no "Basic Usage"? The Advanced Usage section can also use a little reorganization. I kind of like how `django-redis-cache` did it in their docs by separating things into sections like "Django's standard cache api", its own methods, and how to do advanced configuration.

**Comment by niwinz** [comment]
@malefice thanks for the feedback.

In any case, django-redis bord as fork of django-redis-cache because that was unmaintained and contained a lot of bugs that are not permessible to have in production environment. Today the things are changed and seems like the author of django-redis-cache have fixed that issues.

However, django-redis does not forces you use its "extensibility" or advanced stuff. In fact as basic usage is already documented in django documentation, I think repeating the same thing seems like useless. django-redis conforms the django cache backend especification... so with the "quick start" configuration you can use the django's api as is, as is documented in django documentation. This is because no "Basic usage" section.

`get_or_set` method is part of the django cache api, and django-redis conforms it so you can use it out of the box with django-redis... the `nx` parameter is for some one that known redis and want to use its atomicy facilities instead of the django's.

In other words, documenting the django api, that is already documented in django documentation seems like creating redundant information. Maintaining documentation updated take a lot of effort and I think that having that documented in django documentation is more than enough.

**Comment by malefice** [comment]
Like I said, I was new, so I am not so familiar with the history of the packages, and frankly, I did not know better as well. Hahaha! So for documentation, it does not have to repeat everything the Django docs already say, but something like "all the methods from Django's cache api are also available for use, and you may refer to Django's docs here" would have sufficed. Just a suggestion man to help out newbies like I was. Hahaha!

**Comment by niwinz** [comment]
For me it was pretty evident because of "redis cache backend for django" statement in the main description (something that does not implement the django cache backend interface can't be called _cache backend for django_). 

But it seems that I'm wrong and for facilitate things I'll add your suggestions to the documentation, thank you very much for the suggestion ;)

**Comment by Stephane-Ag** [comment]
Where was this added? I still can't find any comparisons.
